data/io.elementary.installer.gresource.xml

```diff
@@ -8,4 +8,8 @@
     <file alias="io.elementary.installer.caution.svg" compressed="true" preprocess="xml-stripblanks">caution.svg</file>
     <file alias="io.elementary.installer.emblem-downloads.svg" compressed="true" preprocess="xml-stripblanks">emblem-downloads.svg</file>
```

Comment:
You moved these icons under the /io/elementary/installer/icons prefix, so maybe we can remove these lines?

Reply:
Oops yes you're right. Thanks for catching that!